services/adapter.ts

```diff
@@ -12,8 +12,16 @@ import './buffer-shim';
 // @ts-ignore
 fetch.Promise = Bluebird.Promise;
 
-export const asyncPipeline = Bluebird.Promise.promisify(pipeline);
-export const asyncExecFile = Bluebird.Promise.promisify(child_process.execFile);
-export const asyncAccess = Bluebird.Promise.promisify(fs.access);
-export const asyncUnlink = Bluebird.Promise.promisify(fs.unlink);
-export const asyncWriteFile = Bluebird.Promise.promisify(fs.writeFile);
+// Sadly these need to be manually typed according to
```

Comment:
Ah, I went this route once, but really did not want to have to do this. Whatever.